setup.py

```diff
@@ -8,8 +8,8 @@
 
 PY_VER = sys.version_info
 
-if not PY_VER >= (3, 5):
-    raise RuntimeError("aioodbc doesn't support Python earlier than 3.5")
+if not PY_VER >= (3, 6):
```

Comment:
I do not think this necessary, 3.5 works fine, we just test library with 3.6 on each PR